test/Microsoft.AspNetCore.Mvc.Core.Test/RedirectToRouteResultTest.cs

```diff
@@ -106,6 +107,9 @@ public class RedirectToRouteResultTest
             serviceProvider
                 .Setup(sp => sp.GetService(typeof(ILoggerFactory)))
                 .Returns(NullLoggerFactory.Instance);
+            serviceProvider
+                .Setup(sp => sp.GetService(typeof(RedirectToRouteResultExecutor)))
+                .Returns(new RedirectToRouteResultExecutor(NullLoggerFactory.Instance, factory.Object));
```

Comment:
Suggest removing this mock. Already using the better `ServiceCollection` approach elsewhere in this class.